frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -160,31 +156,16 @@ const getCreateProps = ({ namespace }: { namespace: string }) => {
 export const VirtualMachinesPage: React.FC<VirtualMachinesPageProps> = (props) => {
   const { namespace } = props;
 
-  const resources = [
```

Comment:
isn't the removal of resources going to damage the vm's status and actions?

Reply:
yes, it will, the loaded resources are expected to be passed to the `VMList` component.